app-misc/vcontrold/vcontrold-0.98.4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+man +vclient vsim"
+
+DEPEND="dev-libs/libxml2
+	man? ( dev-python/docutils )"
+RDEPEND="${DEPEND}"
```

Comment:
dev-python/docutils should be strictly a build-time dep. Is it necessary to keep it after the man pages are built?

And, should man page building be optional? Aren't they already made?

Reply:
The manpages are written in RST and converted to troff build-time. So we only need the Docutils if we want to "compile" the manpages and can cheap out this dependency if they are not needed/wanted. So I thought it would be meaningful to make the manpage building optional.